src/commands/enrolRewardProgram.ts

```diff
@@ -22,7 +22,10 @@ import { CommandUtils } from '../service/CommandUtils';
 export default class EnrolRewardProgram extends Command {
     static description = `It enrols the nodes in the rewards program by announcing the enrol transaction to the registration address.  You can also use this command to update the program registration when you change the node public key or server host.
 
-Currently, the only program that can be enrolled post-launch is 'SuperNode'.`;
+Currently, the only program that can be enrolled post-launch is 'SuperNode'.
+
+This is an ONLINE command as it creates the transaction depending on the main account type (simple or multisig), and it announces the transaction to the network.
```

Comment:
```suggestion
This command can only be used in ONLINE mode since it creates the transaction depending on the main account type (simple or multisig), and it announces the transaction to the network.
```